WebRoot/js/zimbraAdmin/accounts/model/ZaAccount.js

```diff
@@ -1721,6 +1730,29 @@ function(by, val) {
     } else {
         this[ZaAccount.A2_autodisplayname] = "FALSE";
     }
+       
+    var defaultRights = ZaNewAdminWizard.getDefaultDARights (this) ;
+    var count=0;
+    if (allGrantsList.length >= 3) {
+      for (var i = 0; i < allGrantsList.length; i++) {
+        if (allGrantsList[i].grantee_id === this.id) {
```

Comment:
```
if (count > 2) { break; }
if (allGrantsList[i].grantee_id === this.id) {
```

Reply:
Moved these changes to https://github.com/Zimbra/zm-delegated-admin-zimlet/pull/8

Reply:
Please close this PR and delete branch

Reply:
closing the PR